h2o-genmodel/build.gradle

```diff
@@ -9,10 +9,10 @@ javadoc {
   options.overview = "src/main/java/overview.html" // relative to source root
 
   doLast {
-     copy {
-       from "src/main/java/doc-files"
-       into "$buildDir/docs/javadoc/doc-files"
-     }
+    copy {
+      from "src/main/java/doc-files"
```

Comment:
Unnecessary formatting changes. Should be removed before merging.

Reply:
Will fix